charts/kyverno/templates/validate.yaml

```diff
@@ -8,6 +8,19 @@
   {{- end }}
 {{- end }}
 
+# Function to check replicaCount of all deployments
+{{- define "replicaCountCheck" }}
+  {{- $value := . }}
+  {{- if eq $value 0 }}
+    {{ fail "Kyverno does not support running with 0 replicas. Please provide a non-zero value." }}
+  {{- end }}
+{{- end }}
+
+{{- template "replicaCountCheck" .Values.replicaCount }}
```

Comment:
call the helper directly in deployment templates

Reply:
Done